daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/Util.scala

```diff
@@ -67,7 +65,7 @@ object Util {
    * operating system
    */
   def path(string: String): Path = {
-    Paths.get(daffodilRoot, string)
+    Paths.get(string)
```

Comment:
Can you remove daffodilRoot from the comment too? Also, while you're in this section can you replace "line separator" with "path separator" in this comment, I think "line" was probably a typo?